examples/stories/keyframes-object-call-expression.tsx

```diff
@@ -144,6 +143,17 @@ export const CssPropTaggedTemplateExpression = (): JSX.Element => (
   <div css={taggedTemplateExpressionCss}>blue to indigo</div>
 );
 
+const callExpressionCss = css({
+  animationDuration: '2s',
+  animationIterationCount: 'infinite',
+  animationName: fadeOut,
+  animationTimingFunction: 'ease-in-out',
+});
+
+export const CssPropCallExpression = (): JSX.Element => (
+  <div css={callExpressionCss}>blue to indigo</div>
+);
+
 export const InlineCssPropObjectCallExpression = (): JSX.Element => (
   <div
     css={{
```

Comment:
This one was meant to be for the css usage, so you should update it to be `css={css({ /* ... */ })}`

Similarly, `ShorthandInlineCssPropObjectCallExpression` should be updated to `<div css={css({ /* ... */ })}>blue to indigo</div>`

As for `CssPropCallExpression`, you can remove it, or have the original examples in here but under the object expression name, i.e. `InlineCssPropObjectExpression` and `CssPropObjectExpression`